.github/workflows/nightly.yml

```diff
@@ -31,117 +36,157 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: wlan-testing
+        ref: staging-wifi-2047
+
     - name: Checkout LANforge scripts
       uses: actions/checkout@v2
       with:
         path: wlan-lanforge-scripts
         repository: Telecominfraproject/wlan-lanforge-scripts
 
+    - name: import LANforge scripts
+      working-directory: wlan-testing
+      run: ./sync_repos.bash
+
+    - name: create configuration.py
+      working-directory: wlan-testing
+      run: |
+        cat << EOF > configuration.py
+        ${{ secrets.LAB_CONFIGURATION }}
+        EOF
+
+    - name: create Dockerfile
+      working-directory: wlan-testing
+      run: |
+        cat << EOF > Dockerfile
+        FROM python:3.8
+
+        RUN mkdir ~/.pip
+        RUN echo "[global]" > ~/.pip/pip.conf
+        RUN echo "index-url = https://pypi.org/simple" >> ~/.pip/pip.conf
+        RUN echo "extra-index-url = https://tip-read:tip-read@tip.jfrog.io/artifactory/api/pypi/tip-wlan-python-pypi-local/simple" >> ~/.pip/pip.conf
+        RUN pip3 install pytest==6.2.2 allure-pytest bs4 paramiko xlsxwriter requests pandas influxdb influxdb-client tip-wlan-cloud scp
+        COPY lanforge /wlan-testing/lanforge
+        COPY tests /wlan-testing/tests
+        COPY libs /wlan-testing/libs
+        COPY configuration.py /wlan-testing/tests/configuration.py
+
+        WORKDIR /wlan-testing
+        ENTRYPOINT ["bash"]
+        EOF
+
     # build and push docker image
     - name: docker login
-      shell: bash
       run: docker login ${{ env.DOCKER_SERVER }} -u ${{ env.DOCKER_USER_NAME }} -p ${{ env.DOCKER_USER_PASSWORD }}
     - name: build docker image
-      shell: bash
-      run: docker build -t ${{ env.DOCKER_SERVER }}/cloud-sdk-nightly:pytest-${{ github.run_id }} -f wlan-testing/pytest/Dockerfile .
+      working-directory: wlan-testing
+      run: docker build -t ${{ env.DOCKER_SERVER }}/cloud-sdk-nightly:pytest-${{ github.run_number }} .
```

Comment:
So this means that we can delete the docker file from the repo and have our README.md file make use of generated docker images for the testing repo. Thoughts?

Reply:
I would keep the "how to build" section and a one that explains how to pull the latest prebuilt image.